AC58U编译错误

反馈bug/问题模板，提建议请删除

## 1.关于你要提交的问题

Q：是否搜索了issue (使用 "x" 选择)
* [ X] 没有类似的issue

## 2. 详细叙述

### (1) 具体问题

A：编译失败，目测和  404ec5f “ipq40xx: adjust pacakge” 有关

### (2) 路由器型号和固件版本

A：AC58U IPQ40XX 最新commit

### (3) 详细日志

A：Configuring luci-app-accesscontrol.
 * check_data_file_clashes: Package kmod-ath10k-ct-smallbuffers wants to install file /home/runner/work/lede/lede/build_dir/target-arm_cortex-a7+neon-vfpv4_musl_eabi/root-ipq40xx/lib/modules/4.19.115/ath10k_core.ko
	But that file is already provided by package  * kmod-ath10k-ct
 * check_data_file_clashes: Package kmod-ath10k-ct-smallbuffers wants to install file /home/runner/work/lede/lede/build_dir/target-arm_cortex-a7+neon-vfpv4_musl_eabi/root-ipq40xx/lib/modules/4.19.115/ath10k_pci.ko
	But that file is already provided by package  * kmod-ath10k-ct
 * opkg_install_cmd: Cannot install package kmod-ath10k-ct-smallbuffers.

make[2]: *** [package/install] Error 255
package/Makefile:65: recipe for target 'package/install' failed
make[2]: Leaving directory '/home/runner/work/lede/lede'
package/Makefile:108: recipe for target '/home/runner/work/lede/lede/staging_dir/target-arm_cortex-a7+neon-vfpv4_musl_eabi/stamp/.package_install' failed
make[1]: Leaving directory '/home/runner/work/lede/lede'
make[1]: *** [/home/runner/work/lede/lede/staging_dir/target-arm_cortex-a7+neon-vfpv4_musl_eabi/stamp/.package_install] Error 2
make: *** [world] Error 2
/home/runner/work/lede/lede/include/toplevel.mk:225: recipe for target 'world' failed
##[error]Process completed with exit code 2.

--- a/target/linux/ipq40xx/image/Makefile
+++ b/target/linux/ipq40xx/image/Makefile
@@ -91,8 +91,8 @@ define Device/asus_rt-ac58u
 	KERNEL_INITRAMFS := $$(KERNEL) | uImage none
 	KERNEL_INITRAMFS_SUFFIX := -factory.trx
 	IMAGES := sysupgrade.bin
-	DEVICE_PACKAGES := -kmod-ath10k kmod-ath10k-ct kmod-ath10k-ct-smallbuffers \
-		kmod-usb-ledtrig-usbport
+	DEVICE_PACKAGES := -kmod-ath10k -ath10k-firmware-qca4019 \
+	kmod-ath10k-ct-smallbuffers ath10k-firmware-qca4019-ct kmod-usb-ledtrig-usbport
 endef
 TARGET_DEVICES += asus_rt-ac58u
 
@@ -410,7 +410,8 @@ define Device/zyxel_wre6606
 	DEVICE_DTS_CONFIG := config@4
 	IMAGES := sysupgrade.bin
 	IMAGE/sysupgrade.bin := append-kernel | append-rootfs | pad-rootfs | append-metadata
-	DEVICE_PACKAGES := -kmod-ath10k kmod-ath10k-ct kmod-ath10k-ct-smallbuffers
+	DEVICE_PACKAGES := -kmod-ath10k -ath10k-firmware-qca4019 \
+	kmod-ath10k-ct-smallbuffers ath10k-firmware-qca4019-ct
 endef
 TARGET_DEVICES += zyxel_wre6606
 

